snippets/Google.Pubsub.V1.Snippets/PublisherClientSnippets.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Pubsub.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class PublisherClientSnippets
+{
+    public void ListTopics()
+    {
+        // <ListTopics>
+        PublisherClient client = PublisherClient.Create();
+        // Alternative: use a known project resource name:
```

Comment:
It's tricky - normally comments like this refer to the line below rather than the line above, but I do see your point. More generally, we might want to have a single document about resource names and resource IDs, when to use each, how to convert between them etc - then we don't need it in every snippet. We should work out what we _do_ want to put in each snippet.